rosdep/base.yaml

```diff
@@ -7324,6 +7324,11 @@ sdl2-ttf:
   gentoo: [media-libs/sdl2-ttf]
   nixos: [SDL2_ttf]
   ubuntu: [libsdl2-ttf-dev]
+semgrep:
+  osx:
+    homebrew:
+      packages: [semgrep]
+  pip: [semgrep]
```

Comment:
This is not how the `pip` key works; you'll instead need to split this out for each distribution (debian, ubuntu, etc), and then put a pip key underneath it.

Even then, you should know that you cannot release packages that depend on `pip` keys into ROS.  All of the dependencies must be available via system packages.